stable/traefik/templates/deployment.yaml

```diff
@@ -60,6 +60,16 @@ spec:
           periodSeconds: 10
           successThreshold: 1
           timeoutSeconds: 2
+        {{- if and .Values.acme.enabled .Values.acme.dnsProvider.name }}
+        env:
+        {{- range $k, $v := (index .Values.acme.dnsProvider .Values.acme.dnsProvider.name) }}
```

Comment:
I _think_ Go templates let you use `_` here in place of the `$v` you're not using.

Reply:
Doesn't seem like it:
```
Error: parse error in "traefik/templates/deployment.yaml": template: traefik/templates/deployment.yaml:56: function "_" not defined
```
but I can replace it with `$_`.

Will change.

Reply:
Fixed in https://github.com/kubernetes/charts/pull/1299/commits/2054d09db6f79a85932b51a89928a01158d1035a